src/components/views/rooms/LegacyRoomHeader.tsx

```diff
@@ -0,0 +1,822 @@
+/*
```

Comment:
Many violations of https://github.com/vector-im/element-web/blob/develop/code_style.md#react 11

> A component should only use CSS class names in line with the component name.

> When knowingly using a class name from another component, document it with a [comment](https://github.com/vector-im/element-web/blob/develop/code_style.md#comments).

Reply:
Thank you for raising that. Fixed.